portal/templates/flask_user/flask_user_base.html

```diff
@@ -1,17 +1,14 @@
 {% extends "layout.html" %}
 
 {% block main %}
-
-            {% block content %}{% endblock %}
-
+	{% block content %}{% endblock %}
 {% endblock %}
 <script>
 {% block document_ready %}
```

Comment:
If you're able to, I would prefer this block removed from every template :)

I don't think there's an explicit list of everything included in that block, so once it's added to a template it can be difficult to know if it can be removed.

Reply:
yea, block removed, thank you